### ISSUE TITLE
Emoji in v2 API

### ISSUE THREAD

**Issue by pvonmoradi** [chubin/wttr.in#625]
Shouldn't `v2.wttr.in/prague.png` use emojis by default (like its non-image counterpart)?  
`curl "https://v2.wttr.in/prague"`  
![image](https://user-images.githubusercontent.com/1058151/132694441-d824acf9-5f94-4f61-9acb-45decb889ad4.png)

`curl "https://v2.wttr.in/prague.png"`  

![prague](https://user-images.githubusercontent.com/1058151/132695278-2c3fc749-df5d-498c-b841-ee0203a8e04d.png)

**Comment by chubin** [comment]
Yes, it seems to be a regression. The emojis were rendered properly in the png view before. Accepted.
